src/model/quality.rs

```diff
@@ -0,0 +1,49 @@
+use combine::{attempt, choice, optional, parser::char::string, Parser, Stream};
+use strum_macros::{Display, EnumIter};
+use Quality::*;
```

Comment:
This is hoisting all of the enum cases defined below to being top-level names?

Reply:
Yeah